File: src/main/java/org/eln2/sim/mna/component/InterSystem.java
package org.eln2.sim.mna.component;

/*
NOTE: DO NOT IMPORT MINECRAFT CODE IN THIS CLASS
EXTEND IT INSTEAD IN THE org.eln.nbt DIRECTORY
 */

public class InterSystem extends Resistor {

    public static class InterSystemDestructor {
        boolean done = false;
    }

    @Override
    public boolean canBeReplacedByInterSystem() {
        return true;
    }
}

File: src/oldsim/electrical/interop/InterSystem.kt
package org.eln2.oldsim.electrical.interop

import org.eln2.oldsim.electrical.mna.component.Resistor
import org.eln2.oldsim.electrical.mna.state.State

/*
NOTE: DO NOT IMPORT MINECRAFT CODE IN THIS CLASS
EXTEND IT INSTEAD IN THE org.eln.nbt DIRECTORY
 */

open class InterSystem: Resistor {

    constructor()
    constructor(aPin: State?, bPin: State?): super(aPin, bPin)

    override fun canBeReplacedByInterSystem(): Boolean {
        return true
    }
}